docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,62 @@
+---
+title: .NET API Analyzer
+description: Learn how the .NET API Analyzer can help detecting deprecated APIs and platform compatibility issues.
+author: oliag
+ms.author: mairaw
+ms.date: 12/11/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+---
+
+# .NET API Analyzer
+
+The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for APIs with different platforms and detecting calls to deprecated APIs. It can be useful for any developers of the .NET family at any stages of development.
+
+API Analyzer comes as [NuGet package](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after referencing it in a project, it automatically starts monitoring the code and squiggles problematic API usage. You can get suggestions on possible fixes by clicking on the light bulb, which also includes the ability to suppress the warnings.
```

Comment:
comma between "after" and "an"
after referencing it --> after you reference it
I'm not enamored of squiggles as a verb. I would change it to "It automatically monitors the code and indicates problematic API usage."
For the last sentence, "You can also get suggestions on possible fixes by clicking on the light bulb, and you can suppress the warnings."